specs/signature-specification.md

```diff
@@ -177,29 +135,31 @@ Signed attributes/claims are additional metadata apart from the payload, which a
 - Claims that MUST be processed by a verifier MUST be marked as critical. Some claims may be optional and critical, i.e. they MUST be processed by a verifier only if they are present.
 - Claims which are informational and do not influence signature verification MUST NOT be marked critical.
 
-Notary v2 requires the signature envelope to support the following signed attributes/claims.
+Notary Project requires the signature envelope to support the following signed attributes/claims.
 
 #### Standard attributes
 
-- **Signing Scheme** (critical): A REQUIRED claim that defines the [Notary v2 Signing Scheme](./signing-scheme.md) used by the signature. This attribute dictates the rest of signature schema - the set of signed and unsigned attributes to be included in the signature. Supported values are `notary.x509` and `notary.x509.signingAuthority`.
+- **Signing Scheme** (critical): A REQUIRED claim that defines the [Notary Project Signing Scheme](./signing-scheme.md) used by the signature. This attribute dictates the rest of signature schema - the set of signed and unsigned attributes to be included in the signature. Supported values are `notary.x509` and `notary.x509.signingAuthority`.
 - **Signing Time**: A claim that indicates the time at which the signature was generated. Though this claim is signed by the signing key, it’s considered unauthenticated as a signer can modify local time and manipulate this claim. More details [here](#signing-time). This claim is REQUIRED and only valid when signing scheme is `notary.x509` .
 - **Authentic Signing Time** (critical): The authenticated time at which the signature was generated. This claim is REQUIRED and only valid when signing scheme is `notary.x509.signingAuthority` . More details [here](#signing-time).
```

Comment:
Thanks @toddysm I will create a new issue for adding more information for `Authentic Signing Time`, since this doc mainly focuses on fixing naming issue. And the content for `Authentic Signing Time` is not trivial.

Reply:
Both `Authentic Signing Time` and `Signing Time` are explained in detail in linked section of `Signing Time`. I added additional text for `Authentic Signing Time`